service/src/main/java/uk/gov/hmcts/reform/fpl/service/orders/generator/C43ChildArrangementOrderDocumentParameterGenerator.java

```diff
@@ -0,0 +1,134 @@
+package uk.gov.hmcts.reform.fpl.service.orders.generator;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.fpl.config.LocalAuthorityNameLookupConfiguration;
+import uk.gov.hmcts.reform.fpl.enums.C43OrderType;
+import uk.gov.hmcts.reform.fpl.enums.DocmosisTemplates;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.event.ManageOrdersEventData;
+import uk.gov.hmcts.reform.fpl.model.order.Order;
+import uk.gov.hmcts.reform.fpl.service.orders.docmosis.C43ChildArrangementOrderDocmosisParameters;
+import uk.gov.hmcts.reform.fpl.service.orders.docmosis.DocmosisParameters;
+import uk.gov.hmcts.reform.fpl.service.orders.generator.common.OrderMessageGenerator;
+
+import java.util.List;
+
+import static org.apache.commons.lang3.StringUtils.isEmpty;
+
+@Component
+@RequiredArgsConstructor(onConstructor_ = {@Autowired})
+public class C43ChildArrangementOrderDocumentParameterGenerator implements DocmosisParameterGenerator {
+    private final OrderMessageGenerator orderMessageGenerator;
+
+    public static final String WARNING = "Where a Child Arrangements Order is in force and the arrangements "
```

Comment:
I believe "WARNING" short be more descriptive of what it holds. 
When picking variable names, try to find a balance thats descriptive enough that someone can look at it and have a good idea of what it contains, without having so many words it defeats the point of a variable.

"The name of a variable, function or class should answer all big questions. It should tell you why it exists, what it does and how it does it" - Clean Code pg 18